docs/interpreter/spark-interpreter-k8s.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: page
+title: "Apache Spark Interpreter for Apache Zeppelin on Kubernetes"
+description: "Apache Spark is a fast and general-purpose cluster computing system. It provides high-level APIs in Java, Scala, Python and R, and an optimized engine that supports general execution engine. This interpreter runs on the https://github.com/apache-spark-on-k8s/spark version of Spark"
+group: interpreter
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# How to run Zeppelin Spark notebooks on a Kubernetes cluster
+
+<div id="toc"></div> 
+
+## Prerequisites
+
+The following tools are required:
+
+ - Kubernetes cluster & kubectl 
+
+    For local testing Minikube can be used to create a single node cluster: https://kubernetes.io/docs/tasks/tools/install-minikube/ 
+
+ - Docker https://kubernetes.io/docs/tasks/tools/install-minikube/ 
+
+    This documentation uses a pre-built Spark 2.2 Docker images, however you may also build these images as described here: https://github.com/apache-spark-on-k8s/spark/blob/branch-2.2-kubernetes/resource-managers/kubernetes/README.md 
+
+## Checkout Zeppelin source code
+
+Checkout the latest source code from https://github.com/apache/zeppelin then apply changes from the [Add support to run Spark interpreter on a Kubernetes cluster](https://github.com/apache/zeppelin/pull/2637) pull request.
+
+## Build Zeppelin
+- `./dev/change_scala_version.sh 2.11`
+- `mvn clean install -DskipTests -Pspark-2.2 -Phadoop-2.4 -Pyarn -Ppyspark -Pscala-2.11` 
+
+
+## Create distribution
+- `cd zeppelin-distribution`
+- `mvn org.apache.maven.plugins:maven-assembly-plugin:3.0.0:single -P apache-release`
+
+## Create Zeppelin Dockerfile in Zeppelin distribution target folder
+```
+cd {zeppelin_source}/zeppelin-distribution/target/zeppelin-0.8.0-SNAPSHOT
+cat > Dockerfile <<EOF
+FROM banzaicloud/spark-base:v2.2.0-k8s-1.0.179
```

Comment:
It is better to not include any company in apache project doc. It is better to use apache account, BTW how does apache spark do this ?

Reply:
ASF INFRA has support for publishing docker images - Apache Zeppelin in fact has this process setup for its official docker image release

Reply:
Changed doc to use kubespark images (currently available), this needs to be updated once they will push out the official images.